File: fizzbuzz/fizzbuzz.c
#include <stdio.h>

int main()
{
  int i=1;
  for (i=1; i<=15; i++) {
    if (i % 3 == 0) {
      printf("Fizz");
    }
    if (i % 5 == 0) {
      printf("Buzz");
    }
    if (i % 3 != 0 && i % 5 != 0){
      printf("%d", i);
    }
    printf(" ");
  }
  printf("\n");
  return 0;
}

File: fizzbuzz/fizzbuzz.hs
fizzbuzz :: Int -> String
fizzbuzz x 
  | x `mod` 3 == 0 && x `mod` 5 == 0 = "FizzBuzz"
  | x `mod` 3 == 0 = "Fizz"
  | x `mod` 5 == 0 = "Buzz"
  | otherwise = show x

runfb :: Int -> [String]
runfb x =
  let
    n = x
  in [fizzbuzz a | a <- [1..n]]

main :: IO()
main = do
  putStr $ unwords 
         $ runfb 15
